pkg/controller/controllercmd/builder.go

```diff
@@ -175,6 +177,14 @@ func (b *ControllerBuilder) WithInstanceIdentity(identity string) *ControllerBui
 	return b
 }
 
+// WithEventRecorderOptions allows to override the default Kubernetes event recorder correlator options.
+// This is needed if the binary is sending a lot of events.
+// Using events.DefaultOperatorEventRecorderOptions here makes a good default for normal operator binary.
+func (b *ControllerBuilder) WithEventRecorderOptions(options record.CorrelatorOptions) *ControllerBuilder {
+	b.eventRecorderOptions = options
```

Comment:
I expected to set it to the recommended options by default. Is that the case?

Reply:
I guess https://github.com/openshift/library-go/pull/777/files#diff-ff937793ae2933db156923ac8eaebbd8R271 is doing this, right?